helix-term/src/application.rs

```diff
@@ -676,8 +676,8 @@ impl Application {
                 let notification = match Notification::parse(&method, params) {
                     Ok(notification) => notification,
                     Err(err) => {
```

Comment:
Let's keep the old block as a fallthrough and specifically match on `Err(helix_lsp::Error::Unhandled)` here: this error is only benign if we fail to parse the notification because we don't handle it yet. It might be malformed in other ways that we should log as an error

Reply:
Good call. I made the change.